src/pkcs11/pkcs11-spy.c

```diff
@@ -844,6 +844,26 @@ C_DecryptInit(CK_SESSION_HANDLE hSession, CK_MECHANISM_PTR pMechanism, CK_OBJECT
 	enter("C_DecryptInit");
 	spy_dump_ulong_in("hSession", hSession);
 	fprintf(spy_output, "pMechanism->type=%s\n", lookup_enum(MEC_T, pMechanism->mechanism));
+	switch (pMechanism->mechanism) {
+	case CKM_RSA_PKCS_OAEP:
+		if (pMechanism->pParameter != NULL) { /* XXX assuming RSA-OAEP parameter */
```

Comment:
This comment is not needed anymore, since the mechanism is already checked. I would remove it to avoid confusing further readers.

Reply:
Addressed, thanks!